src/components/SudokuGrid/Row.js

```diff
@@ -1,17 +1,29 @@
 import Cell from "./Cell";
 
-export default function Row({ rowData, rowNumber, rowMask, onCellInRowClick }) {
+export default function Row({
+  rowData,
+  rowNumber,
+  rowMask,
+  onCellInRowClick,
+  activeCellID,
+}) {
   return (
     <div className="Row">
       {rowData.map((cell, columnNumber) => {
         const isEditable = rowMask[columnNumber] === 0;
+        const isSelected =
+          activeCellID.length === 2 &&
+          rowNumber === activeCellID[0] &&
+          columnNumber === activeCellID[1];
```

Comment:
Abstract this as well. Just call a function that gives you the result. For better readability.